.changeset/big-hotels-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@eth-optimism/contracts': minor
```

Comment:
This should be a patch release as we do minor releases with each regenesis and a major release with bedrock